tests/testthat/test-step-hcai-missing.R

```diff
@@ -0,0 +1,135 @@
+context("Checking recipe step hcai-missing")
+
+library(healthcareai)
+library(tibble)
+library(caret)
+library(recipes)
+
+# Setup ------------------------------------------------------------------------
+# set seed for reproducibility
+set.seed(7)
+# build dataset
+n <- 300
+d <- tibble(id = 1:n,
+                 playerID = sample(1:9, size = n, replace = TRUE),
+                 level = sample(1:12, size = n, replace = TRUE),
+                 world = sample(1:8, size = n, replace = TRUE),
+                 character = sample(c("Mario", "Luigi"), size = n,
+                                    replace = TRUE),
+                 suit = sample(c("Fire Flower", "Raccoon", "Frog", "P Wing"),
+                                    size = n, replace = TRUE)
+                 )
+
+# target
+d["is_goomba"] <- ifelse( (d["world"] - 2 * d["level"] - 1) > 0, "Y", "N")
```

Comment:
Just FYI, with `tibble` you can reference already-created variables in the initialization.